integration/testrunner/grpcrunner/grpc.go

```diff
@@ -48,6 +54,8 @@ type GRPC struct {
 	BigQueryClient *bigquery.Client
 }
 
+const privateKey = "-----BEGIN EC PRIVATE KEY-----\nMHcCAQEEIITZ4357UsTCbhxXu8w8cY54ZLlsAIJj/Aej9ylb/ZfBoAoGCCqGSM49\nAwEHoUQDQgAEhBWj8vw5LkPRWbCr45k0cOarIcWgApM03mSYF911de5q1wGOL7R9\nN8pC7jo2xbS+i1wGsMiz+AWnhhZIQcNTKg==\n-----END EC PRIVATE KEY-----"
```

Comment:
Add a comment link to public key(s).
Add a comment on public key(s) and link them back here.